.github/workflows/android.yml

```diff
@@ -13,6 +13,8 @@ jobs:
             - uses: actions/checkout@v2
 
             - uses: actions/setup-ruby@v1
+              with:
+                  ruby-version: '2.7'
```

Comment:
Should we lock down all the other usages of `setup-ruby`?

Reply:
I would assume that the same error could occur on any of the workflows that are using setup-ruby and bundler. Probably a good idea to do the same in all the other workflows.